src/views/Tenant/TenantView.js

```diff
@@ -165,9 +165,9 @@ const Tenant = () => {
       comp: <div />,
     },
     {
-      key: "unit",
-      label: "Unit",
-      value: property.unit,
+      key: "num_units",
+      label: "Total Units",
+      value: property.num_units,
```

Comment:
If I'm not mistaken, shouldn't this be the `unitNum` value of the lease?  Looking at page 10 of the [figma](https://drive.google.com/file/d/1YqboQogczYm1HkyRqEtVSzeQ61T9hWU2/view), I'd interpret this as the individual unit number connected to the tenant, rather than the number of units contained in the property.

Reply:
Ah you're right! I'll update that. thanks for catching it!